package.json

```diff
@@ -1,7 +1,26 @@
 {
+  "name": "purescript-elmish",
+  "license": "MIT",
+  "description": "A PureScript implementation of The Elm Architecture (TEA)",
+  "author": {
+    "name": "Fyodor Soikin",
+    "email": "fyodor@collegevine.com",
+    "url": "https://github.com/fsoikin"
+  },
+  "contributors": [
+    {
+      "name": "Daniel Gasienica",
+      "email": "daniel.gasienica@collegevine.com",
+      "url": "https://github.com/gasi"
+    },
+    {
+      "name": "Abby Sassel",
+      "url": "https://github.com/sassela"
+    }
+  ],
```

Comment:
Moved these fields here because `spago.dhall` doesn't support them. 

@gasi @sassela please indicate your preference for whether or not to include you and what level of detail to provide.

Reply:
I'd love to be included — thanks! Those details are fine.